components/expenses/Expense.js

```diff
@@ -468,17 +516,49 @@ class Expense extends React.Component {
                     {canMarkExpenseAsUnpaid && <MarkExpenseAsUnpaidBtn refetch={this.props.refetch} id={expense.id} />}
                     {canApprove && <ApproveExpenseBtn refetch={this.props.refetch} id={expense.id} />}
                     {canReject && <RejectExpenseBtn refetch={this.props.refetch} id={expense.id} />}
+                    {canDelete && (
+                      <DeleteButton
+                        bg="red.600"
+                        buttonStyle="primary"
+                        onClick={() => this.setState({ showDeleteExpenseModal: true })}
+                      >
+                        <FormattedMessage id="expense.delete.btn" defaultMessage="Delete" />
+                      </DeleteButton>
+                    )}
                   </div>
                 </div>
               )}
             </div>
           )}
+          {this.state.error && (
+            <div className="error" data-cy="errorMessage">
```

Comment:
We should prefer styled-components for new developments. This one can easily be replaced by a 
```suggestion
            <P color="red.500" data-cy="errorMessage">
```